src/java.base/share/classes/jdk/internal/math/FDBigInteger.java

```diff
@@ -90,111 +55,96 @@
             long[] long5pow = {
```

Comment:
Would it be worthwhile to make these arrays `@Stable`?

In the future, might be a candidate for StableConstant/LazyValue functionality.

Reply:
> Would it be worthwhile to make these arrays `@Stable`?
> 
> In the future, might be a candidate for StableConstant/LazyValue functionality.

PS If each array is read only once, then perhaps the other data structures would be better initialized in a loop.